app/models/node.rb

```diff
@@ -404,8 +404,7 @@ def has_power_tag(key)
 
   # returns the value for the most recent power tag of form key:value
   def power_tag(tag)
-    tids = Tag.includes(:node_tag)
-              .references(:community_tags)
+    tids = Tag.includes(%i(node_tag community_tags))
```

Comment:
Hey @Tlazypanda I believe this is causing an error because we do not have a relation between community_tags and tags . I think we might have to leave it as is

Reply:
OK, closing due to this! Thank you!